logic/common/level_writer.py

```diff
@@ -0,0 +1,69 @@
+'''Utility class for processing and outputting an XML file'''
+
+#import _common.__base as b
+from logic.common import file_utils as b
```

Comment:
I would just write this as "from logic.common import file_utils".

No need to shorthand it to "fu". file_utils is short enough and informative. As "fu" it is confusing and less readable.